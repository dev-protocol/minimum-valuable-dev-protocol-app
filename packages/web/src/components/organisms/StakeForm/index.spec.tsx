import React from 'react'
import { render } from '@testing-library/react'
import { StakeForm } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('@dev/graphql')

describe(`${StakeForm.name}`, () => {
  test('Snapshot', () => {
    const component = render(<StakeForm propertyAddress="propertyAddress" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
