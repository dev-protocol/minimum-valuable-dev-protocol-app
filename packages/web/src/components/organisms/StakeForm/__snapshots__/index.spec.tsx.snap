// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StakeForm Snapshot 1`] = `
<body>
  <div>
    <div
      class="w1tfby-1 jGLtYs"
    >
      <div
        class="w1tfby-2 htxnGK"
      >
        <label
          for="stake"
        >
          Stake
        </label>
        <span
          class="ant-input-group-wrapper ant-input-group-wrapper-lg ant-input-search ant-input-search-large ant-input-search-with-button w1tfby-0 frIYqB"
        >
          <span
            class="ant-input-wrapper ant-input-group"
          >
            <span
              class="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
            >
              <input
                class="ant-input ant-input-lg"
                id="stake"
                type="number"
                value=""
              />
              <span
                class="ant-input-suffix"
              >
                <span>
                  DEV
                </span>
                <button
                  class="ant-btn ant-btn-primary ant-btn-sm sc-1ttu0x8-0 pssBW"
                  type="button"
                >
                  <span>
                    Max
                  </span>
                </button>
              </span>
            </span>
            <span
              class="ant-input-group-addon"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg ant-input-search-button"
                type="button"
              >
                <span>
                  Stake
                </span>
              </button>
            </span>
          </span>
        </span>
        <div
          class="ant-card ant-card-bordered w1tfby-3 gaYbxX"
        >
          <div
            class="ant-card-head"
          >
            <div
              class="ant-card-head-wrapper"
            >
              <div
                class="ant-card-head-title"
              >
                Estimated Annual Reward
              </div>
            </div>
          </div>
          <div
            class="ant-card-body"
          >
            <p>
              0
               DEV
            </p>
          </div>
        </div>
      </div>
      <div
        class="w1tfby-2 htxnGK"
      >
        <label
          for="withdraw"
        >
          Withdraw
        </label>
        <span
          class="ant-input-group-wrapper ant-input-group-wrapper-lg ant-input-search ant-input-search-large ant-input-search-with-button w1tfby-0 frIYqB"
        >
          <span
            class="ant-input-wrapper ant-input-group"
          >
            <span
              class="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
            >
              <input
                class="ant-input ant-input-lg"
                id="withdraw"
                type="number"
                value=""
              />
              <span
                class="ant-input-suffix"
              >
                <span>
                  DEV
                </span>
                <button
                  class="ant-btn ant-btn-primary ant-btn-sm sc-1ttu0x8-0 pssBW"
                  type="button"
                >
                  <span>
                    Max
                  </span>
                </button>
              </span>
            </span>
            <span
              class="ant-input-group-addon"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg ant-input-search-button"
                type="button"
              >
                <span>
                  Withdraw
                </span>
              </button>
            </span>
          </span>
        </span>
        <div
          class="ant-card ant-card-bordered w1tfby-3 gaYbxX"
        >
          <div
            class="ant-card-head"
          >
            <div
              class="ant-card-head-wrapper"
            >
              <div
                class="ant-card-head-title"
              >
                Withdrawable Amount
              </div>
            </div>
          </div>
          <div
            class="ant-card-body"
          >
            <p>
              5000
               DEV
               
              (0 + 5000)
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
